### PR TITLE
Fix/docs

### PR DESCRIPTION
**Pull request by quasiben**
Doc changes.  I mixed both style and typo fixes.  @jcrist if you'd like I can separate these changes in to separate PRs -- this might make style changes easier to discuss and/or ignore

### CODE REVIEW

File: docs/source/quickstart.rst [hunk 1]
```diff
@@ -29,8 +29,8 @@ Dask-Gateway can be installed with ``conda`` or ``pip``. It's composed of two pa
     $ pip install dask-gateway dask-gateway-server
 
 
-Start the gateway server
-------------------------
+1) Start the gateway server
+---------------------------
```
Comment (jcrist) [inline docs/source/quickstart.rst]: Why the numbering?

**Comment by quasiben** [comment]
Please push whatever you like

File: docs/source/quickstart.rst (hunk 1 above)
Comment (quasiben) [reply docs/source/quickstart.rst]: My thought was the numbering clearly indicated steps: 1, 2, etc.  I can remove them if don't like them.

**Comment by jcrist** [comment]
Apologies for letting this linger so long. I've fixed the merge conflicts and pushed a few small tweaks. This looks good to me, thanks! Merging.

**Pull request merged**